main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,292 @@
+Waiting for proposal to pass (status=PROPOSAL_STATUS_VOTING_PERIOD)
+Waiting for proposal to pass (status=PROPOSAL_STATUS_VOTING_PERIOD)
+Waiting for proposal to pass (status=PROPOSAL_STATUS_VOTING_PERIOD)
+Waiting for proposal to pass (status=PROPOSAL_STATUS_VOTING_PERIOD)
+Waiting for proposal to pass (status=PROPOSAL_STATUS_VOTING_PERIOD)
+Waiting for proposal to pass (status=PROPOSAL_STATUS_VOTING_PERIOD)
+Import the following mnemonic into Keplr:
+survey thank matrix joke trim more make gossip spread yellow unfold under cash beach harsh fire blush achieve oak swamp pluck clock rocket leg
+
+The resulting address should be: agoric1xe269y3fhye8nrlduf826wgn499y6wmnv32tw5
+
+make: Leaving directory '/workspace/contract'
+Done in 34.95s.
+```
 
-![Dapp card store ui - needs approval](./assets/must-enable-dapp.png)
+![Pasting the mnemonic phrase](./assets/041.png)
 
-1. Navigate back to [http://127.0.0.1:8000](http://127.0.0.1:8000) and accept the Dapp's request to connect to your wallet.
+Give your new wallet a name and a password.
 
-![agoric wallet - Dapp approval prompt](./assets/accept-dapp-connection.png)
+![Name the newly imported wallet](./assets/042.png)
 
-2. Navigate back to [the Dapp](http://localhost:3000) and it should load the baseball cards to bid on
+Click "Save".
 
-![Dapp card store ui - loading cards](./assets/card-store-ui.png)
+![Save](./assets/043.png)
 
-## Use the Dapp to bid on and buy a baseball card
+Starting the Dapp
+To start the UI for the sample dapp, run the `yarn start:ui` command. Note the localhost link that appears on your terminal window. Open this link in your browser.
 
-1. In the Dapp, you should be able to click on a baseball card to `BID` on it in an action. Enter `Bid ammount` to submit an offer to buy the card.
+![Running the `yarn start:ui` command](./assets/044.png)
 
-   ![Bid on Card](./assets/bid-on-card.png)
+![Open the link in your browser](./assets/045.png)
 
-1. In the wallet, `Approve` the `Proposed` offer to bid on a card.
+From the browser UI, click the "Connect Wallet" button to connect your Keplr wallet. You will be asked to approve this connection.
 
-   ![Proposed Offer](./assets/proposed-offer.png)
+![Connecting your wallet](./assets/046.png)
 
-1. In the wallet, the offer will be in a `Pending` state while the auction for the card to complete. The auction takes up to 300 seconds.
+Once your wallet is connected, click on the "Make Offer" button to purchase 3 properties. Approve the transaction in your Keplr wallet.
 
-   ![Pending Offer](./assets/pending-offer.png)
+![Making an offer](./assets/047.png)
 
-1. In the wallet, your offer will transition to an `Accepted` state when the auction ends. Your `cardStore.Card` purse will now contain a card.
+When the transaction is complete you will notice some IST has been debited from your wallet, and you are the owner of three new properties.
 
-   ![Accepted Offer](./assets/accepted-offer.png)
+![Finished transaction](./assets/048.png)
```

Comment:
This screenshot still has the "off brand" place names. I trust you'll get to that soon. It's one of my last critical comments.

Many of my other comments are probably not critical and I can argue about them in later PRs :)